inteq valor and haste changes

Jedi-Toothpaste opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Fixes a few things regarding the Valor-Class that were unintentionally overwrote by #3508 and fixes, finally, the fact the landing pad was never centered. Alongside a few QoL Fixes to boot on the Valor and Haste.
On-top of this, the Haste-Class has been made bigger, given window shutters and we've taken a stasis bed from the Valor itself, and moved it onto the Haste; this was required to make the shuttle centered, and will make transporting patients easier with a larger shuttle.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

### **Check the drop-downs below for change comparisons.**
<details><summary>Changes to the Landing Pad</summary>
<p>

![image](https://github.com/user-attachments/assets/d2d9e557-57fd-4658-bab4-a3341a79d92d)

![image](https://github.com/user-attachments/assets/d30b7570-4a3f-41d7-a8bf-03e8009a2d2e)
</p>
</details> 

<details><summary>Changes to the Haste</summary>
<p>

![image](https://github.com/user-attachments/assets/3e848227-d815-4cea-8b5c-58b5c7935c8a)

As you can see, the Haste has been made larger, and a stasis bed has been dragged from the medical area to here. 
But, as a cost, the Fourth Defib has been taken from the ship- a tragedy, i'm sure. However, there are still 3 on the Valor itself.

Furthermore, blast-doors have been added to the windows, alongside a button to open and close them, a welcome change i'm sure.

</p>
</details> 

<details><summary>Uncategorized Changes</summary>
<p>

![image](https://github.com/user-attachments/assets/2fea1d53-d052-413d-a386-f35f52b04240)

Removed a second privacy shutters button in the surgery room (was where red circle was)
Added an intercom outside the surgery room for patient use (blue circle)
The keen eye will also notice how the surgery room has a vent and scrubber, that's been added alongside a vent and scrubber in the EVA room.

![image](https://github.com/user-attachments/assets/6ff08c06-1071-4bbf-b316-dea88c344711)

Removed- this light switch, idk how it got here; it can never spawn naturally because it spawns on a null tile, but. It's gone now I suppose???

Alongside this lightswitch, a few lights have been moved around to ensure they still light up the entire hangar.

![image](https://github.com/user-attachments/assets/745cb010-69ae-4881-88dd-8e7dda7418c6)

Finally, the Hangar will have an Air Alarm!!! Rejoice.

</p>
</details> 


## Why It's Good For The Game
Small cleanup, and QoL for the Valor is *good*.

As for the Haste changes, making is symetrical has given it a great time to be remapped to be bigger and more specialised for moving patients, as it is right now it can barely hold a response team, let alone bringing them back. With the extra room no longer will we have to lay on-top of eachother on the floor. Mostly.

Moving the stasis bed helps this, too, and sells the Haste as a shitty box ambulance more than it was.
And as for the windows? It would be nice to not die instantly to one carp chewing the engines as much, bringing it in line with every other shuttle as to having shuttered windows.

Final question you may ask:

> Since you moved stuff from the Valor to the Haste, why not move the Holopad from the mess, too?

Now, this one is a deeply personal feeling of mine, that, when you fuck up as the Haste, saying it into Wideband is infinitely funnier. This is all.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: Air Alarms and Vents to the Valor
change: New Landing Pad, it's now symettrical 
change: Made the haste class bigger, and given it window shutters
change: Moved a stasis bed from the Valor-class to the Haste-class
change: Removed a defibrillator from the Haste-class (valor still spawns with 3 round-start, we don't need more)

/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Jedi-Toothpaste committed:
inteq valor and haste changes

Apogee-dev commented:
i’ll be real i’m not a fan of the thicc haste or the expanded landing pad aesthetically. it was also purposefully off center, i wouldnt call that a fix per se. same for the haste coming with a stretcher instead of a stasis bed. that was a deliberate choice to discourage using the haste itself as a mobile treatment center. the platforms on the landing pad also look pretty bad to me- what ship that isnt some kind of supercarrier’s gonna have enough room in the hangar for a raised landing pad with a shuttle on top? most other ships that have an in-hangar landing pad make it either level (which is to say, the shuttle is landing directly on the hangar deck) or a pit (see: heron, tegu, the WIP anaconda)

Jedi-Toothpaste committed:
fixes the paper

may need to remove the cute paper memo if this doesn't work, I will weep if I have to, however.

Jedi-Toothpaste commented:
> i’ll be real i’m not a fan of the thicc haste or the expanded landing pad aesthetically. it was also purposefully off center, i wouldnt call that a fix per se. same for the haste coming with a stretcher instead of a stasis bed. that was a deliberate choice to discourage using the haste itself as a mobile treatment center. the platforms on the landing pad also look pretty bad to me- what ship that isnt some kind of supercarrier’s gonna have enough room in the hangar for a raised landing pad with a shuttle on top? most other ships that have an in-hangar landing pad make it either level (which is to say, the shuttle is landing directly on the hangar deck) or a pit (see: heron, tegu, the WIP anaconda)

Hi, I will defend the larger Haste-Class as often Valor teams will travel in 3, which takes up 3 of the 5 tiles on the previous sized Haste. It just feels bad on a small shuttle.
Alongside this, the stasis bed is just more so you don't have to bring formaldehyde *all* the time. Plus, what's stopping the Haste doing mobile surgery on a rollerbed? It has a similar surgery-step failure chance that the stasis does, doing surgery on-site is a player quality problem, rather than a map issue.

I will concede the raised platform since I understand the raised platform doesn't fit now I look, I just wanted to look for a way to seperate the pad from the rest of the hangar. Here is my proposed change- but it may just be a case of putting it back to how it was.
![image](https://github.com/user-attachments/assets/d093468d-470c-4d21-98cd-daf72179cc10)

meemofcourse commented:
> Uncategorized Changes

Sure.

> Changes to the Haste

The handrail is cool. That's probably going to be standard for most subshuttles, anyway.

The size change? No. The ambulance is intended to be the smallest possible size (without turning into a pill) in order to fit into the most spaces possible. Making it bigger defeats that. If you want to, you can try putting the gear in a wall locker to open up a tile. By extension, you're going to have to live with the off-set landing pad. I know I put a lot of symmetry in my own ships, but, sorry.

I'm eh on the stasis bed. That seems more fitting for like a slower, larger subshuttle. You're supposed to getting to the place fast, and getting out fast. Realistically, this might not happen too much., but it's what I envisioned when I made this thing. I'd be up for seeing it tested, but as for right now I'd like to keep the roller bed on the ambulance.

Jedi-Toothpaste committed:
stasis bed change

changes stasis bed on the haste back to a roller, and restores the stasis bed on the valor

Jedi-Toothpaste commented:
Actually, what'd be best is just scrapping this pull, since it's no longer fixes to both; i'll reopen a fresh PR with changes learned from here at a later date.
